### PR TITLE
Update server player props article

### PR DESCRIPTION
**Pull request by xxshady**

### CODE REVIEW

File: docs/articles/player/server-player-props.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Used to **get or set** the player's current health.
 
 Health in GTA:V is based on the values `100 - 200`.
 
-200 is the player's maximum health.
+200 is the default player's maximum health (`mp_m_freemode_01` model).
```
Comment (C0kkie) [inline docs/articles/player/server-player-props.md:123]: change to "default freemode player model" or something

Comment (xxshady) [reply docs/articles/player/server-player-props.md]: Changed

Comment (xxshady) [reply docs/articles/player/server-player-props.md]: changed

Comment (xxshady) [reply docs/articles/player/server-player-props.md]: wait, this: `Health in GTA:V is based on the values `100 - 200`.` is incorrect too

Comment (xxshady) [reply docs/articles/player/server-player-props.md]: updated

Comment (C0kkie) [reply docs/articles/player/server-player-props.md]: Can we please remove that mp_m_freemode_01 what i say the whole time xd you would need to mention the female version too if you mention the male one, so just remove the example

Comment (xxshady) [reply docs/articles/player/server-player-props.md]: Can't i just write that its **male** model and not mention female?

Comment (C0kkie) [reply docs/articles/player/server-player-props.md]: I think its better to keep it neutral

**Pull request merged**